### PR TITLE
Graphs

### PR DESCRIPTION
**Pull request by laf**
Please see our [contributing docs](https://github.com/librenms/librenmsv2/blob/develop/docs/General/Contributing.md)

- [x] Have you created any necessary docs?
- [x] Have you created tests for Travis integration?
- [x] Have you used localisation in the WebUI?
- [x] Have you included the proper copyright?

Won't be the best code just yet but gets the basics down and adds some examples of:

- Adding a datatables page without widgets but on a page with other elements.
- Adding graphs with varying options to produce png or js graphs (only device_bits has both options available).
-  Started the device overview page + it's file structure.

image

### CODE REVIEW

**Comment by murrant** [comment]
Missing the file `resources/views/devices/page/graphs.blade.php`

**Comment by laf** [comment]
That page isn't done, I just created the tabs for visual reference. Same with 'Health'

**Comment by murrant** [comment]
Oh, thanks I'll check again.  I had to wait for more graph data to collect as I didn't have much on my test system...

**Comment by murrant** [comment]
Well, I did `composer dump-autoload` and set $config['install_dir'] and it is working :)
I've been hacking on it a bit...

Not sure the helpers approach is the best, but I can understand pushing dealing with those until later.

File: app/Helpers/Rewrite.php [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<?php
+
+function format_proc_descr($descr)
+{
+    $bad_descr = array(
+        '/GenuineIntel:/' => '',
+        '/AuthenticAMD:/' => '',
+        '/Intel(R)/' => '',
```
Comment (Jellyfrog) [inline app/Helpers/Rewrite.php]: Typo? `Intel\(R\)`

File: app/Helpers/Rewrite.php [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+        '/CPU/' => '',
+        '/(R)/' => '',
```
Comment (Jellyfrog) [inline app/Helpers/Rewrite.php]: Typo? `\(R\)`

File: app/Helpers/Rewrite.php (hunk 1 above)
Comment (murrant) [reply app/Helpers/Rewrite.php]: No, these are regex.  For preg_replace.

Comment (Jellyfrog) [reply app/Helpers/Rewrite.php]: yes, which would match the string "_IntelR_", but I'm pretty sure @laf is trying to match "_Intel(R)_", else it makes no sense to include the parentheses

Comment (murrant) [reply app/Helpers/Rewrite.php]: Hmm, this is legacy code, perhaps it has always been broken...

**Comment by laf** [comment]
Thanks murrant :)

**Comment by murrant** [comment]
Thank you great work.

**Pull request merged**